pylint/checkers/typecheck.py

```diff
@@ -88,6 +88,9 @@ class VERSION_COMPATIBLE_OVERLOAD:
 VERSION_COMPATIBLE_OVERLOAD_SENTINEL = VERSION_COMPATIBLE_OVERLOAD()
 
 
+PY310_PLUS = sys.version_info >= (3, 10)
```

Comment:
Found this cool def already:

`from astroid.const import PY310_PLUS`

Reply:
Let's use the one in pylint :)

Reply:
py310 wasn't defined in constants, so I added it there.

Reply:
Let's define it here then: https://github.com/PyCQA/pylint/blob/main/pylint/constants.py#L21